specutils/tests/spectral_examples.py

```diff
@@ -71,22 +76,29 @@ def __init__(self):
                                         flux=self._flux_e2 * u.mJy)
 
         #
-        # Create on spectrum with the same flux but in angstrom units
+        # Create one spectrum with the same flux but in angstrom units
         #
 
         self.wavelengths_AA = self.wavelengths_um * 10000
         self._s1_AA_mJy_e3 = Spectrum1D(spectral_axis=self.wavelengths_AA * u.AA,
                                         flux=self._flux_e1 * u.mJy)
 
         #
-        # Create on spectrum with the same flux but in angstrom units and nJy
+        # Create one spectrum with the same flux but in angstrom units and nJy
         #
 
         self._flux_e4 = (self.base_flux + 400 * np.random.random(self.base_flux.shape)) * 1000000
         self._s1_AA_nJy_e4 = Spectrum1D(spectral_axis=self.wavelengths_AA * u.AA,
                                         flux=self._flux_e4 * u.nJy)
 
 
+        #
+        # Create one spectrum like 1 but with a mask
+        #
+        self._s1_um_mJy_e1_masked = copy(self._s1_um_mJy_e1)  # SHALLOW copy
```

Comment:
maybe comment why shallow is emphasized here?  It's true, but does it matter in context?